cypress/e2e/projectsModule/projectsModuleUsingPOM.cy.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types= "cypress" />
+
+import addProjectPO from "../../page_objects/projectsModulePO/addProject_PO"
+import { getRandomString , getRandomEmail} from "../../../utilites/helper";
+
+describe('Verifying Add Project functionality of projects module', function(){
+
+    beforeEach(function(){
+        cy.login() 
+        addProjectPO.goToProjectModule()
+    })
+    
+    it('Matching searched string from the list if exists', function(){
+        const searchKeyword = 12345
+        cy.intercept('GET', '/api/project').as('loadProjectModule')
+        cy.wait('@loadProjectModule')
+        addProjectPO.elements.searchField().type(searchKeyword).should('have.value', searchKeyword)
+        cy.get('.table tbody tr').should('have.length', 1)
```

Comment:
add an extra assertion by looping over the tr and making sure if td contains that search key word